static/game/draw.js

```diff
@@ -277,20 +279,28 @@ export function roundedRect(
   context.save();
   topLeftVec = topLeftVec.add(getCameraOffset());
 
+  // If it is a rounded rectangle
+  let rounded = true;
+
   /** @type {{tl: number, tr: number, br: number, bl: number}} */
   let corners;
   if (typeof borderRadius === "number") {
-    corners = {
-      tl: borderRadius,
-      tr: borderRadius,
-      br: borderRadius,
-      bl: borderRadius
-    };
+    if (borderRadius !== 0) {
+      corners = {
+        tl: borderRadius,
+        tr: borderRadius,
+        br: borderRadius,
+        bl: borderRadius
+      };
+    } else {
+      rounded = false;
+    }
   } else {
     corners = borderRadius;
   }
 
   if (lineWidth === undefined) lineWidth = 0;
+  // TODO do we really want to be adjusting by the line width?
```

Comment:
I think so. The issue arises when you have a thick lineWidth, so a box that's supposed to be 100 by 100 pixels can actually end up looking much larger than that because of the 20 pixels of border.